distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -80,20 +79,20 @@ org.codice.ddf.system.organization=${organization.name}
 org.codice.ddf.system.registry-id=
 
 #
-# Solr Client Configuration
+# Solr Provider Configuration
+# Uncomment or comment the properties based on the desired configuration:
 #
-# Uncomment the properties based on the desired configuration:
 
-# For Solr Server
-solr.data.dir=_DO_NOT_EXPAND_${karaf.home}/solr/server/solr
+# Solr Server Provider
+solr.start=true
 solr.client=HttpSolrClient
+solr.data.dir=_DO_NOT_EXPAND_${karaf.home}/solr/server/solr
 solr.http.port=8994
 solr.http.protocol=https
-
-# Do NOT update the solr.http.url when quick installing on a remote headless server.
+solr.mem=2g
```

Comment:
❓Did we lower the amount of memory allocated to the DDF Java process to account for the increased amount of memory allocated to Solr?

Reply:
No, I do not believe we made any changes. 
I would not recommend making any changes until we have some operational definition of "enough memory".  For example:

* We take a particular data set and make it the "reference set".  
* Next, we define an operation(s) on the data. For instance, "ingest all data". 
* Then we define failure criteria like "Java Out of Memory exception" or "takes longer than an hour".
* Next, we experiment. We keep ratching down the memory available to DDF and Solr until the failure occurs. 
* Finally, we publish the "minimum memory requirements" as the lowest values that successfully ingested the reference data.